lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -26,7 +25,10 @@ function multiplyFractions(node) {
     arg => Node.Type.isFraction(arg));
   const hasPolynomialTerms = node.args.some(
     arg => Node.PolynomialTerm.isPolynomialTerm(arg));
-  if (!atLeastOneFraction || hasPolynomialTerms) {
+  const hasConstantNumeratorAndPolynomialInDenominatorTerms = node.args.some(
+    arg => hasConstantNumeratorAndPolynomialInDenominator(arg));
```

Comment:
Serdar taught me to write long descriptive names like that, hehe. I was hesitant at first but now I like it!